extensions/jira/deployment/src/main/java/org/apache/camel/quarkus/component/jira/deployment/JiraProcessor.java

```diff
@@ -55,8 +58,27 @@ NativeImageResourceBuildItem nativeImageResources() {
     }
 
     @BuildStep
-    ReflectiveClassBuildItem registerForReflection() {
-        // Required by org.apache.camel.component.jira.consumer.WatchUpdatesConsumer
-        return new ReflectiveClassBuildItem(true, false, Issue.class.getName());
+    ReflectiveClassBuildItem registerJiraClassesForReflection(CombinedIndexBuildItem combinedIndex) {
+        IndexView index = combinedIndex.getIndex();
+        String[] modelClasses = index.getKnownClasses().stream()
+                .map(ci -> ci.name().toString())
+                .filter(n -> n.startsWith(JIRA_MODEL_PACKAGE))
+                .sorted()
```

Comment:
Out of curiosity, is there any specific need to sort them ?

Reply:
not really.

Reply:
So maybe it could be removed if we consider it's better for build time and readibility.